### ISSUE TITLE
How can the application be translated?

### ISSUE THREAD

**Issue by Y-PLONI** [MahStudio/WinGo-Maps#13]
Thank you very much!
I came across the app now and it looks great.
After I translate, I will examine it more deeply.

**Comment by NGame1** [comment]
This app has stopped working from a long time ago.
I need a new API Key to be able to recover it.
And I can't get this from Google

**Comment by Y-PLONI** [comment]
The app works for me! It also detected my location, and shows me the map!
![image](https://github.com/user-attachments/assets/8d47e239-6f50-4c7e-b761-462693c5721f)

**Comment by NGame1** [comment]
Yes but the most important part is the routing. You can't search for nearby places and route to them.

**Comment by Y-PLONI** [comment]
you are right Only now, after you wrote it, I saw.
Why can't you get an API key from Google?
Can I get a free personal key and push it?

**Comment by NGame1** [comment]
I'm from Iran and due to U.S sanctions I can't have a credit card.
Google asks for a credit card and some other things that are not supported in Iran. 
So if you can get the API key yes you can make app work once again after years.
But first take a very look at Google API key laws and don't push your API key to repo.
If you want share it with me I will try to update the Windows Store app directly
And I really appreciate your kind help.

**Comment by Y-PLONI** [comment]
How interesting! I'm from Israel!!!
How are you now? Calm, or is there fear? I am very interested!
Regarding the API:
I would appreciate it if you could explain to me exactly what I should be looking at, and whether the process of releasing the API is completely free, or I may commit to a payment.

**Comment by NGame1** [comment]
Nice to know, 
Hope you are doing well. 
Things are mostly normal, and really hope countries stop their reactions to eact other. 

You need to first add your billing profile to your Google Account in https://console.cloud.google.com
Then create a new project and enable the services below in the image and then create a new Credentials As API Key type and the API key has no restrictions as Google don't support Windows applications so we can't limit it.

![image](https://github.com/user-attachments/assets/69491451-d849-4251-ae0e-8aabd208709d)

About if it can commit you a payment or not, I'm not completely sure. 
But I think for your personal use it should be free.
In the following document Google says it's free upto 200$ for every month. But how fast you can reach it, I have no idea. 
https://mapsplatform.google.com/pricing/

**Comment by Y-PLONI** [comment]
I join Tikva, praying to God that peace abounds!
I will check about the API.
Maybe it is possible to give a place in the software to enter a personal API key?

**Comment by NGame1** [comment]
Currently no. You should add your personal API Key into code as mentioned in the readme. 
https://github.com/MahStudio/WinGo-Maps?tab=readme-ov-file#build-and-running-the-code

**Comment by Y-PLONI** [comment]
@NGame1 
I tried compiling the software but was unsuccessful.  
The software was built for Windows 10, using an older version — and the same goes for all its dependencies.  
I attempted to update it, but I keep running into errors again and again...  
Could you please help?  
Kindly update the project so it works properly in 2025.  
Thank you so much!
